views.py

```diff
@@ -1,47 +1,86 @@
 from flask import Blueprint, render_template, request, redirect, url_for
+import json
+import os
+
 
 main = Blueprint('main', __name__)
 
-items = []
+
 @main.route('/')
 def index():
-	return render_template('homepage.html', items=items)
+	#with open('list.json') as file:
+	#	items = json.load(file)
+	with open('lists.json') as file:
+		try:
+			lists = json.load(file)
+		except:
+			lists = ''
+	return render_template('homepage.html', lists=lists)
 
-@main.route('/list')
+@main.route('/list', methods=['POST'])
 def list():
-	return render_template('list.html', items=items)
-
-@main.route('/remove', methods=['POST'])
-def list_remove():
-	list_remove = request.form.get('removes')
-	if list_remove:
-		items.remove(f'{list_remove}')
-	return render_template('homepage.html', items=items)
+	name = request.form.get('chooselist')
+	with open('/Users/austin/src/web_app/lists/' + name + '.json') as file:
```

Comment:
`/Users/austin/src/web_app/lists/` is re-used a few times & could be broken out into a variable.

```
listDir = '/Users/austin/src/web_app/lists/'

...
... stuff
...

# Here on L23
with open(listDir + name + '.json') as file:
```



It could also probably be simplified from an absolute path to a relative path. Assuming that this project is living in `~/src/web_app1/`, it would be something like `../web_app/lists/`. `..` means up 1 directory. 

Or maybe make a new directory in this project and add it to .gitignore so that the json files don't annoy you in git.